### PR TITLE
chore: exclude commit messages in CI from linting

### PR DESCRIPTION
**Pull request by MkMan**
Not sure why but the CI build started [failing](https://github.com/aesop/aesop-gel/runs/4860166580?check_suite_focus=true#step:10:222) recently when trying to commit because the commit message body had long lines. It might be due to the version increase done in #652.

This change should exclude commit messages made in the CI from commit linting.

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -4,4 +4,5 @@ module.exports = {
     'header-max-length': [2, 'always', 101],
     'scope-case': [0],
   },
+  ignores: [(message) => message.includes('[skip ci]')],
```
Comment (MkMan) [inline commitlint.config.js:7]: The commit messages made in the CI follow the template defined here https://github.com/aesop/aesop-gel/blob/main/.releaserc#L16

**Review by cyrusxzw — APPROVED** [review]

File: commitlint.config.js (hunk 1 above)
Comment (MkMan) [reply commitlint.config.js:7]: Configuration reference [here](https://commitlint.js.org/#/reference-configuration?id=javascript)

**Review by JasonPingAesop — APPROVED** [review]

**Review by maxlew — APPROVED** [review]
👍

File: commitlint.config.js (hunk 1 above)
Comment (JasonPingAesop) [reply commitlint.config.js:7]: Commitlint added new rules in config-conventional.js
[here](https://github.com/conventional-changelog/commitlint/blob/5a5f6419b6e89a32ef3faa80dc229e08a1e218af/%40commitlint/config-conventional/index.js#L5)
That's why it has limitation on body text message. We can set `body-max-line-length` in our commitlint.config.js

**Pull request merged**